mixer/pkg/runtime2/listener_test.go

```diff
@@ -18,10 +18,11 @@ import (
 	"errors"
 	"reflect"
 	"testing"
-	"time"
 
 	"github.com/gogo/protobuf/proto"
 
+	"sync"
```

Comment:
file is not properly goimport'ed, `synch` should be between `reflect` and `testing`
standard is:

```
  go packages (alphabetized)
  <space>
  github packages (alphabetized)
  <space>
  istio packages (alphabetized)
```

Reply:
Done. PTAL